functions/two_sum.py

```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
+        for j in range(i + 1, len(n)):
```

Comment:
Nice work! I'm concerned about the time  complexity of this function? Do think there is a way to lower it?